library/src/main/java/org/apache/apex/malhar/lib/dedup/TimeBasedDedupOperator.java

```diff
@@ -186,7 +186,7 @@ public void deactivate()
   @Override
   protected void putManagedState(Object tuple)
   {
-    ((ManagedTimeUnifiedStateImpl)managedState).put(getTime(tuple), getKey(tuple), new Slice(new byte[0]));
+    ((ManagedTimeUnifiedStateImpl)managedState).put(getTime(tuple), getKey(tuple), new Slice(null));
```

Comment:
@vrozov The test "DeduperTimeBasedPOJOImplTest" is failing if we dont do this change as a part of upgrade. Apex-core 3.6.0 is using netlet 1.3.0 which throws IllegalArgumentException from Slice().